tests/suppliers_factory.py

```diff
@@ -1,3 +1,26 @@
+
 """
 Test Factory to make fake objects for testing
-"""
\ No newline at end of file
+"""
+import factory
```

Comment:
Meet this error when running "nosetests"

`from .suppliers_factory import SupplierFactory
  File "/vagrant/tests/suppliers_factory.py", line 5, in <module>
    import factory
ModuleNotFoundError: No module named 'factory'`

I ran the "vagrant up" and "vagrant provision" commands and it seems it collected the factory-boy package. Is this the right way to reset the environment after you add new packaged requirements?

Reply:
When you run vagrant up, all python packages defined in requirement.txt will NOT be reinstalled. It's only done in the VM provision process. So when we update our python package suit, you need to run `pip install -r requirements.txt` to install the new package.